infra/Gemfile

```diff
@@ -0,0 +1,6 @@
+source 'https://rubygems.org'
+
+gem 'chef'
+gem 'knife-solo'
+gem 'berkshelf'
```

Comment:
berks is awesome, its dependencies for "i want to install x" basically, so just say the name of what you want in `Berksfile` and it will import it